app/src/main/java/com/weatherxm/ui/photoverification/gallery/PhotoGalleryViewModel.kt

```diff
@@ -34,9 +35,9 @@ class PhotoGalleryViewModel(
 
     fun getLocalPhotosNumber() = photos.filter { it.localPath != null }.size
 
-    fun addPhoto(path: String) {
+    fun addPhoto(path: String, photoSource: PhotoSource?) {
```

Comment:
In both calls of `addPhoto`, `photoSource` isn't `null`, so no need to make it nullable here. Let's remove the `?` until we need it.